### ISSUE TITLE
Globally apply "dontQuote:true"

### ISSUE THREAD

**Issue by levibostian** [hiddentao/squel#214]
My situation:  

* I use squel in my nodejs/expressjs REST APIs for great ways to generate Postgres queries without having to write them raw. (Thank you so much. I love this library!)
* I use [pg](https://www.npmjs.com/package/pg) to do all of my database interaction from my model classes. 
* The pg module has [support for preventing SQL Injection](https://github.com/brianc/node-postgres/wiki/FAQ#8-does-node-postgres-handle-sql-injection). It does this through me providing parameterized queries:
```
client.query("INSERT INTO user(name) VALUES($1)", ["'; DROP TABLE user;"], function (err, result) {
  // This code will not drop the user table. It sanitizes the array of parameters I provide. 
});
```
* So, I am using Squel a tiny different so that I can provide all my user input via parameters into the pg module query. To take advantage of the SQL injection support, I am using the parameter array in my queries:
```
db.query(squel.insert().into("users").set('email', '$1').set('username', '$2').set('created', '$3').toString(), [emailAddress, username, createdDate], function(err, result) {
        return done(err);
    });
```
*(Instead of having `.set('email', emailAddress)` I am doing `.set('email', '$1')`. That is really the only difference I need to do.)*

* However, the pg module will take my array of parameters: `[emailAddress, username, createdDate]` and **add quotes around them automatically** if a variable is a string. Squel **also** puts **quotes around params automatically** and because it is reading '$1', '$2', '$3' as options it thinks those are all strings and quote them. So I have 2 sets of quotes in my queries for my parameters causing postgres errors. 

* I have fixed this issue by telling Squel not to use quotes in each set function call: 
```
db.query(squel.insert().into("users").set('email', '$1', {dontQuote: true}).set('username', '$2', {dontQuote: true}).set('created', '$3', {dontQuote: true}).toString(), [emailAddress, username, createdDate], function(err, result) {
        return done(err);
    });
```
There. Now everything works great. My queries are running successfully. 

But I really don't want to have to include {dontQuote: true} a hundred times in my application. Is there any way that I can tell Squel dontQuote once globally?

**Comment by levibostian** [comment]
I am sorry, I figured it out. Squel already has great support for this built in. 

I did not [check out the parameters](https://hiddentao.github.io/squel/#parameters) section of the website.  To be honest all I have been doing on the website is click on the buttons on the top header (which are handy) and simply never scrolled down. 

Added postgres flavor, use `.toParam().text` and I am good to go. 

Love this library even more. Keep rocking it.
